src/mpyl/steps/deploy/k8s/chart.py

```diff
@@ -481,6 +486,34 @@ def _get_job(self) -> Job:
             raise AttributeError("deployment.kubernetes.job field should be set")
         return job
 
+    def _create_sealed_secret_env_vars(
+        self, secret_list: list[KeyValueProperty]
+    ) -> list[V1EnvVar]:
+        return [
+            V1EnvVar(
+                name=e.key,
+                value_from=V1EnvVarSource(
+                    secret_key_ref=V1SecretKeySelector(
+                        key=e.key, name=self.release_name, optional=False
+                    )
+                ),
+            )
+            for e in secret_list
+        ]
+
+    def _create_secret_env_vars(self, secret_list: list[KeyValueRef]) -> list[V1EnvVar]:
```

Comment:
there must be a way these functions can be combined

Reply:
I can make an interface to map a KeyValueRef and KeyvValueProperty to V1EnvVar but then this implementation moves to `project.py` which is weird I think